analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -105,9 +114,9 @@ protected override void Initialize(SonarAnalysisContext context)
         protected override ControlFlowGraph CreateCfg(SemanticModel model, SyntaxNode node, CancellationToken cancel) =>
             node.CreateCfg(model, cancel);
 
-        protected override void AnalyzeSonar(SyntaxNodeAnalysisContext context, bool isTestProject, bool isScannerRun, SyntaxNode body, ISymbol symbol)
+        protected override void AnalyzeSonar(SyntaxNodeAnalysisContext context, IEnumerable<DiagnosticDescriptor> diagnosticsToRun, bool isTestProject, bool isScannerRun, SyntaxNode body, ISymbol symbol)
         {
-            var enabledAnalyzers = SonarRules.Where(x => x.SupportedDiagnostics.Any(descriptor => IsEnabled(context, isTestProject, isScannerRun, descriptor))).ToArray();
+            var enabledAnalyzers = SonarAnalyzer.Where(x => diagnosticsToRun.Any(descriptor => IsEnabled(context, isTestProject, isScannerRun, descriptor))).ToArray();
```

Comment:
Depending on the "useSonarCfg" we need to execute all or just the non-migrated Sonar rules. The "Base" decides about that.